app/classifier/tasks/drawing/markings-renderer.cjsx

```diff
@@ -35,15 +35,14 @@ module.exports = React.createClass
 
   render: ->
     skippedMarks = 0
-
     <g>
-      {for annotation in @props.classification?.annotations ? []
+      {for annotation in @props.classification?.annotations ? []      
         annotation._key ?= Math.random()
         isPriorAnnotation = annotation isnt @props.annotation
         taskDescription = @props.workflow.tasks[annotation.task]
         if taskDescription.type is 'drawing'
           <g key={annotation._key} className="marks-for-annotation" data-disabled={isPriorAnnotation || null}>
-            {for mark, i in annotation.value when parseInt(mark.frame) is parseInt(@props.frame)
+            {for mark, i in annotation.value when @props.workflow.configuration.multi_image_clone_markers or parseInt(mark.frame) is parseInt(@props.frame)
```

Comment:
Sorry, one other change. The default for `props.workflow` is `null`, so this should be `@props.workflow?.configuration.multi_image_clone_markers` to avoid breaking errors when `workflow.configuration` doesn't exist.